spec/compute_spec.rb

```diff
@@ -74,6 +67,13 @@ def initialize(_args); end
       end
     end
 
+    module Fog
+      module BothWays
+        extend Provider
+        service(:compute, "Compute")
+      end
+    end
```

Comment:
Same thing here. The code you've changed should not break this tests, should it?